Makefile

```diff
@@ -762,3 +762,33 @@ include tests/integration/tests.mk
 .PHONY: benchcheck
 benchcheck:
 	bin/perfcheck.sh
+
+#-----------------------------------------------------------------------------
+# Target: istio-ecosystem installer
+#-----------------------------------------------------------------------------
+
+export ISTIO_INSTALLER?=${GOPATH}/src/istio-ecosystem/istio-installer
```

Comment:
Mine is in `/src/github.com/istio-ecosystem/istio-installer`. I don't mind moving mine if I am the one with a weird path, but we should make sure it is whatever is standard (which I am not sure)

Reply:
Nah, that's better. That's what I meant to type, just messed up when moving stuff around.